hack/manifests/testing/parodos-patch.yaml

```diff
@@ -19,7 +19,7 @@ data:
   DATASOURCE_USERNAME: "parodos"
   DATASOURCE_PASSWORD: "parodos"
   SPRING_PROFILES_ACTIVE: "dev"
-  NOTIFICATION_SERVER_ADDRESS: "notification-service.default.svc.cluster.local"
+  NOTIFICATION_SERVER_URL: "http://notification-service.default.svc.cluster.local:8080"
```

Comment:
@gciavarrini pls be aware of this change. 
this is changed by your open PR - https://github.com/parodos-dev/parodos/pull/376/files#diff-be2e48c86fd33f58e083bafa29850d09d6b6b5ac31c5e6371bd8d686a1bfdd3c